app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt

```diff
@@ -104,6 +104,8 @@ class CustomDrugEntrySheet : BaseBottomSheet<
 
   override fun createEffectHandler(viewEffectsConsumer: Consumer<CustomDrugEntryViewEffect>) = effectHandlerFactory.create(this).build()
 
+  override fun viewEffectsHandler() = CustomDrugEntryViewEffectHandler(this)
```

Comment:
I think there is a typo here, the method name is just `viewEffectHandler`

Reply:
🤔 Umm seems right to me. @msasikanth It's an overridden method from `BaseBottomSheet`. Or, do you mean we want to change the name in `BaseBottomSheet` to `viewEffectHandler`?

Reply:
Ah, interesting. The `BaseScreen` and `BaseBottomSheet` has different function name. Yep, this looks fine.